src/coreclr/pal/inc/pal_mstypes.h

```diff
@@ -195,12 +195,7 @@ extern "C" {
 // they must be either signed or unsigned) and we want to be able to use
 // __int64 as though it were intrinsic
 
-#ifdef HOST_64BIT
```

Comment:
There is an error after deleting this on LoongArch64.

Reply:
@jkotas

Reply:
> @jkotas Why define the `ULONG64` as `unsigned long long` ? While define `SIZE_T` as `unsigned long` ?

Thanks
I had noted #77268

Reply:
this has been special cased to MacOS only in this PR: https://github.com/dotnet/runtime/pull/77268, so guessing that should solve your issue?

Reply:
> this has been special cased to MacOS only in this PR: #77268, so guessing that should solve your issue?

Yes, it's ok for LA64 now.
Thanks.